src/components/Addbook.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
+
+function Addbook() {
```

Comment:
- [ ]  Kindly try to follow the latest ES6 syntax guideline. Try to use the arrow function instead.